equinix/helper_test_test.go

```diff
@@ -0,0 +1,43 @@
+package equinix_test
+
```

Comment:
This file bugs me too, but again, it's a conscious choice to minimize the code changes here.  Note that `helper_test.go` must have that name in order to be loaded for the tests, so this file is called `helper_test_test.go`; one alternative would be to just drop these tests into `helper_test.go`.

Reply:
Wouldn't be a bad idea just to move it into `helper_test.go`. There's not a lot here.